includes/wsuwp-shortcode-fundselector.js

```diff
@@ -605,6 +617,14 @@ jQuery.extend(jQuery.ui.autocomplete.prototype.options, {
 	open: function(event, ui) {
 		jQuery(this).autocomplete("widget").css({
             "width": (jQuery(this).width() + "px")
-        });
+		});
+
+		var length = jQuery(this).autocomplete("widget")[0].childNodes.length;
+		var $lastNode = jQuery(this).autocomplete("widget")[0].childNodes[length - 1];
+
+		if ($lastNode.innerText.indexOf("narrow down your search".toUpperCase()) != -1) {
+			jQuery('#' + $lastNode.id).prop('disabled', true);
+			jQuery('#' + $lastNode.id).addClass('narrow-search', true);
```

Comment:
When you remove the custom class from the CSS, you won't need the `addClass`. Keep the `disabled` though.
The jQuery method had some lag between the time the list was shown and when the styling was applied that you could see. The CSS method is much faster.